Update features/attributify.md to be more professional (#155)
windicss/docs: features/attributify.md

   dark=...
   ```

> > And of course, you can also **mix them**, but it isn't recommended, as this reduces code clarity.

Reviewer: ```suggestion
> And of course, you can **mix them** together, but we **don't recommend** it because it reduces the clarity of your code.
```
Just a suggestion, what do you think?
